fury/text_tools.py

```diff
@@ -0,0 +1,844 @@
+"""A set of tools to deal with texture atlas and freetype fonts.
+
+The objects: TextureAtlas, TextureFont and TextureGlyph was obtained
+from the example provided by Nicolas P. Rougier at here
+
+https://github.com/rougier/freetype-py/blob/09a664d23850a7ba8cc208443b1c4f96f21116c6/examples/texture_font.py
+which has the follwoing copyright
+
+freetype-py is licensed under the terms of the new or revised BSD license, as
+follows:
+
+Copyright (c) 2011-2014, Nicolas P. Rougier
+All rights reserved.
+
+We fixed some bugs in those objects.
+
+"""
+
+import sys
+import numpy as np
+from scipy import ndimage
+from PIL import Image
+import pickle
+import os
+from os.path import join as pjoin
+
+try:
+    import freetype as ft
+    _FREETYPE_AVAILABLE = True
+except ImportError:
+    _FREETYPE_AVAILABLE = False
+
+import fury
+from fury.data.fetcher import fury_home
+
+_FONT_PATH_DEFAULT = f'{fury.__path__[0]}/data/files/font_atlas'
+_FONT_PATH_TTF = f'{fury.__path__[0]}/data/files/'
+_FONT_PATH_USER = pjoin(fury_home, 'font_atlas')
+
+
+def norm(img, vmin=0, vmax=255):
+    if vmin == 0:
+        img = np.interp(img, (img.min(), img.max()), (vmin, vmax))
+        # img /= np.max(img)
+        # img *= vmax
+        # img[img > 0.75] = 0.
+    elif vmin > 0:
+        # pass
+        img = np.interp(img, (img.min(), img.max()), (vmin, vmax))
+    img = img.astype(np.uint8)
+    return img
```

Comment:
I am surprised that we do not have this function already somewhere or neither in `numpy`.  Can you check it? 
Also, this is an utility function, I recommend to move it somewhere else

Reply:
I've created that because we can have different strategies to normalize the SDF.  This different strategies controls how the field will decay which directs impact the redering result. For example, 
```python
        img = np.interp(img, (img.min(), img.max()), (vmin, vmax))
```
or 
```python
img /= np.max(img)
img *= vmax
img[img > threshold] = 0.
 ```       
@filipinascimento can we discuss which behavior we will choose?